tests/jballerina-unit-test/src/test/resources/test-src/expressions/let/let-expression-test.bal

```diff
@@ -362,6 +362,11 @@ function testLetWithClass() {
     assert(6, foo2.m);
 }
 
+function testLetWithXMLTemplateExpression() {
+    xml x = xml `<Books>${xml `<Book>"${let int y = 5 in y}"</Book>`}</Books>`;
+    assert("<Book>\"5\"</Book>", (x[0]/*).toString());
```

Comment:
Shall we try to avoid using `toString()` and assert the actual value? We may lose relevant type info when we assert the string representation.